typer/_typing.py

```diff
@@ -448,25 +367,16 @@ def is_callable_type(type_: Type[Any]) -> bool:
     return type_ is Callable or get_origin(type_) is Callable
 
 
-if sys.version_info >= (3, 7):
```

Comment:
I think you inverted this condition. The else block applies if 3.6 was being used.

Reply:
Right, good catch, thanks!